### DISCUSSION TITLE
How to do emissive textures?

### DISCUSSION THREAD

**Discussion by Murasaki2142** [MoreMcmeta/core#113] [Help Wanted]
i have my .properties file set up and my texture has an emissive map, but it doesnt glow ingame?

**Comment by soir20** [comment]
There's documentation about how to use OptiFine .properties files here (which also works with MoreMcmeta): https://optifine.readthedocs.io/emissive_textures.html

Documentation about MoreMcmeta's additional JSON format: https://github.com/MoreMcmeta/core/wiki/User-Docs:-Emissive-Format

There's a sample emissive texture in this demo resource pack (and any emissive OptiFine packs should also work: https://www.curseforge.com/minecraft/texture-packs/moremcmeta-demo

What object are you trying to make emissive? Are you using any rendering mods? Perhaps there's an incompatibility causing it to not glow
